style.css

```diff
@@ -486,9 +498,89 @@ img {
   border-radius: 8px;
   margin-top: 10px;
   margin-left: 5px;
+  width: 327px;
+  height: 170px;
 }
 
 #get-in-touch-button {
   margin-top: 12px;
   margin-left: 5px;
 }
+
+@media only screen and (max-width: 768px) {
+  .desktop-toolbar {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .mobile-toolbar > nav,
+  .icon {
+    display: none;
+  }
+
+  .desktop-toolbar > nav {
+    font-family: 'Poppins', sans-serif;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 10em;
+    margin: 1em 1em 1em 1em;
+  }
+
+  .desktop-toolbar-secondary {
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+    font-family: 'Poppins', sans-serif;
+    font-size: 2rem;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 1.4rem;
+    margin: 1.2rem 5rem;
+    align-items: center;
+    align-content: space-between;
+    justify-content: space-between;
+    color: #344563;
+    list-style-type: none;
+  }
+
+  .icon {
+    display: block;
+    margin: auto;
+  }
+
+  .work {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
- [ ] Kindly make sure your projects section is centred and looks as close as possible to the Figma design 🙏 
- [ ] Kindly make sure your paragraph and text doesn't look squished together please give it the appropriate padding and spacing 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/186783906-ec1170d4-f6bc-42ee-9654-762e7ba370fa.png)  |  ![image](https://user-images.githubusercontent.com/74814780/186783995-f133aa10-30a8-43c9-a6ca-6db6cde3397c.png)

Reply:
Done!